### ISSUE TITLE
Symbols should not be allowed as operands of arithmetic operations.

### ISSUE THREAD

**Issue by DanielRosenwasser** [Microsoft/TypeScript#1750]
```TypeScript
function f() {
    var symbol = Symbol();

    try {
        symbol + "";
        return false;
    }
    catch(e) {}

    try {
        symbol + 0;
        return false;
    }
    catch(e) {}

    return true;
}
```

We should give an error on `symbol + ""` and `symbol + 0`.
